.prettierignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
can you remove this file and add some test cases for this feature? others LGTM

Reply:
yep sure. I forgot to remove.prettierignore
I need it on my branchto not mess your code styling.
I will put test cases and will be ready with final version till the end of the week.

Reply:
I think I'm ready with my changes. I've added tests. They passed successfully :)
And .prettierignore was removed.